lib/resin-lint.ts

```diff
@@ -130,7 +143,13 @@ const runLint = function(resinLintConfig: ResinLintConfig, paths: string[], conf
 	const scripts = findFiles(resinLintConfig.extensions, paths);
 
 	if (resinLintConfig.lang === 'typescript') {
-		linterExitCode = lintTsFiles(scripts, config);
+		let prettierConfig: prettier.Options | undefined;
+		if (resinLintConfig.prettierCheck) {
+			prettierConfig = parseJSON(prettierConfigPath)  as prettier.Options;
```

Comment:
Super nit: double space before `as` (haha, are we not running prettier on this repo itself?)